boxfile.yml

```diff
@@ -22,7 +26,7 @@ data.db:
 web.main:
   start:
     nginx: nginx -c /app/config/nginx.conf
-    puma: bundle exec puma -C /app/config/puma.rb
+    puma: bundle exec rails s -b 0.0.0.0 -p 8080
```

Comment:
Any particular reason you pulled out puma here?

Reply:
It still boots and uses puma, as intended, I believe this to be the more conical way to boot the rails app. 

Part of this was to pass in the binding and port settings, which were set before via a monkey patching hack that is no longer supported with Rails 5.1